gdcdictionary/schemas/sample.yaml

```diff
@@ -505,7 +505,6 @@ properties:
       - Frozen
       - Unknown
       - Not Reported
-      - Not Allowed To Collect
   sample_type:
```

Comment:
Do we want to remove `sample_type` in lieu of `composition` and `tumor_type`?

Reply:
Yeah ideally we can get rid of it. It's still used in active projects, but I personally think its overloaded and the decomposition handles it better.

Reply:
And then would probably need to make at least `composition` required?

Reply:
Yes. I'd say probably tissue_type as well but at the minimum composition replaces sample_type.

Reply:
Ok, let's make them both required then.

Reply:
Removed `sample_type`; required `composition` and `tissue_type`.